ansible_collections/arista/avd/molecule/eos_cli_config_gen/intended/configs/vlan-interfaces.cfg

```diff
@@ -90,17 +90,17 @@ interface Vlan75
    multicast ipv6 boundary ff00::/16 out
    multicast ipv6 boundary ff01::/16 out
    multicast ipv4 static
-   ipv6 virtual-router address 1b11:3a00:22b0:1000::1
```

Comment:
Key-id order
```
interface Vlan43
   description SVI Description
   no shutdown
   ipv6 dhcp relay destination a0::2 vrf TEST local-interface Loopback44 link-address a0::4
   ipv6 address a0::1/64
   isis authentication key-id 2 algorithm sha-512 key 0 password
   isis authentication key-id 3 algorithm sha-512 rfc-5310 key 0 password1
   isis authentication key-id 1 algorithm sha-1 key 0 password level-1
```

```
interface Vlan43
   description SVI Description
   ipv6 dhcp relay destination a0::2 vrf TEST local-interface Loopback44 link-address a0::4
   ipv6 address a0::1/64
   isis authentication key-id 1 algorithm sha-1 key 7 qZQzXEhWASbvrmcvHhTCZw==
   isis authentication key-id 2 algorithm sha-512 key 7 qZQzXEhWASbvrmcvHhTCZw==
   isis authentication key-id 3 algorithm sha-512 rfc-5310 key 7 yN8FIWN8ZbkSvVqiF90Gfw==
```

Reply:
```
interface Vlan44
   description SVI Description
   no shutdown
   ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-address a0::7
   ipv6 dhcp relay destination a0::8
```

```
interface Vlan44
   description SVI Description
   ipv6 dhcp relay destination a0::8
   ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-addr
ess a0::7
```

Reply:
```
interface Vlan50
   description IP NAT Testing
   ip nat source static 3.0.0.1 4.0.0.1
   ip nat source dynamic access-list ACL2 pool POOL2
   ip nat destination static 1.0.0.1 2.0.0.1
   ip nat destination dynamic access-list ACL1 pool POOL1
```

```
interface Vlan50
   description IP NAT Testing
   ip nat destination static 1.0.0.1 2.0.0.1
   ip nat source static 3.0.0.1 4.0.0.1
   ip nat destination dynamic access-list ACL1 pool POOL1
   ip nat source dynamic access-list ACL2 pool POOL2
   isis authentication mode text rx-disabled level-2
   isis authentication key 7 bpM/wnochsWdunkFr0o/Bg== level-2
```

Reply:
About the isis authentication key-id order the logic is like it will first sort the isis both type level key-ids then lervel-1 and then level-2 so we will see the order difference here.
@ClausHolbechArista @gmuloc

Reply:
> ```
> interface Vlan44
>    description SVI Description
>    no shutdown
>    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-address a0::7
>    ipv6 dhcp relay destination a0::8
> ```
> 
> ```
> interface Vlan44
>    description SVI Description
>    ipv6 dhcp relay destination a0::8
>    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-addr
> ess a0::7
> ```

We are already sorting base on address but EOS sorting base on vrf and none vrf destinations

Reply:
> ```
> interface Vlan44
>    description SVI Description
>    no shutdown
>    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-address a0::7
>    ipv6 dhcp relay destination a0::8
> ```
> 
> ```
> interface Vlan44
>    description SVI Description
>    ipv6 dhcp relay destination a0::8
>    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-addr
> ess a0::7
> ```

We alrready have sorting base on address in j2 temp but EOS use another criteria like vrf and none vrf destinations

Reply:
> ```
> interface Vlan50
>    description IP NAT Testing
>    ip nat source static 3.0.0.1 4.0.0.1
>    ip nat source dynamic access-list ACL2 pool POOL2
>    ip nat destination static 1.0.0.1 2.0.0.1
>    ip nat destination dynamic access-list ACL1 pool POOL1
> ```
> 
> ```
> interface Vlan50
>    description IP NAT Testing
>    ip nat destination static 1.0.0.1 2.0.0.1
>    ip nat source static 3.0.0.1 4.0.0.1
>    ip nat destination dynamic access-list ACL1 pool POOL1
>    ip nat source dynamic access-list ACL2 pool POOL2
>    isis authentication mode text rx-disabled level-2
>    isis authentication key 7 bpM/wnochsWdunkFr0o/Bg== level-2
> ```

this ip nat is comes from interface-ip-nat.j2 so we can check there only

Reply:
> > ```
> > interface Vlan44
> >    description SVI Description
> >    no shutdown
> >    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-address a0::7
> >    ipv6 dhcp relay destination a0::8
> > ```
> > 
> > 
> >     
> >       
> >     
> > 
> >       
> >     
> > 
> >     
> >   
> > ```
> > interface Vlan44
> >    description SVI Description
> >    ipv6 dhcp relay destination a0::8
> >    ipv6 dhcp relay destination a0::5 vrf TEST source-address a0::6 link-addr
> > ess a0::7
> > ```
> 
> We are already sorting base on address but EOS sorting base on vrf and none vrf destinations

Updated the sorting for ipv6 dhcp relay destination